application.go

```diff
@@ -344,11 +357,44 @@ func (r *Application) AddEnv(name, value string) *Application {
 	return r
 }
 
+// AddEnvSecret adds an environment variable secret to the application
+//		name:	the name of the variable
+//		secret:	the name of secret defined in the application's Secrets
+func (r *Application) AddEnvSecret(name, secret string) *Application {
```

Comment:
Can we define this function after `AddSecret` to match the order of the underlying structs defined above?